src/language-css/printer-postcss.js

```diff
@@ -203,47 +198,45 @@ function genericPrint(path, options, print) {
 
       if (isLessParser(options)) {
         if (node.mixin) {
-          return concat([
+          return [
             path.call(print, "selector"),
             node.important ? " !important" : "",
             isTemplatePlaceholderNodeWithoutSemiColon ? "" : ";",
-          ]);
+          ];
         }
 
         if (node.function) {
-          return concat([
+          return [
             node.name,
-            concat([path.call(print, "params")]),
+            [path.call(print, "params")],
```

Comment:
This one is a sigle `Doc`, should not use `concat`.

I'm going to fix this first, it's easier to detect before we change `concat()` to `array`.

Reply:
#9994